src/redux/api.js

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios';
+
+const baseURL = 'https://api.covid19tracking.narrativa.com';
+
+const API = {
+  getAllCountries: (success) => {
+    axios.get(`${baseURL}/api/countries`)
+      .then((res) => {
+        success(res);
+      });
+  },
+
+  getAllCountriesData: (date, success) => {
+    axios.get(`${baseURL}/api/${date}`)
+      .then((res) => {
+        success(res);
+      });
+  },
+};
```

Comment:
- [ ] <b>OPTIONAL:</b> This is not best practice. here, you are recreating an instance of axios every time you are want to use it which is not good for memory management, this can be avoided by doing this: 
   -  create a file and call it `axios.js` may be. and then on that fle,
      ```js
        import axios from 'axios';
        
       const baseURL = 'https://api.covid19tracking.narrativa.com';
       export default axios.create({
          baseURL,
       });
       ```
     - And then in your API file, instead of importing axios from `axios`, you import if from `./axios.js` the file we have created above.